jaas/command/src/main/java/org/apache/karaf/jaas/command/WhoamiCommand.java

```diff
@@ -0,0 +1,76 @@
+package org.apache.karaf.jaas.command;
```

Comment:
...and now, the part where I hold my breath and see what Jenkins has to say this time.